cmd/thermal-recorder/config.go

```diff
@@ -75,7 +80,12 @@ func (conf *MotionConfig) Validate() error {
 	return nil
 }
 
+type rawUploaderConfig struct {
+	DeviceName string `yaml:"device-name"`
```

Comment:
Given there's no special pre-processing required for `device-name` there's no need to have a separate `rawUploaderConfig`. Just read straight into `UploaderConfig` (but remember to move the struct tags)